docs/Users-Guide.md

```diff
@@ -316,3 +348,114 @@ Using the advances search, a JSON can be passed to Timesketch
+
+#### Browser Search Analyzer
+
+The browser search analyzer takes URLs usually resevered for browser search queries and extracts the search string.
+
+It will also tell in a story:
+
+- The top 20 most commonly discovered searches
+- The domains used to search
+- The most common days of search
+- And an overview of all the discovered search terms
+
+#### Browser Timeframe Analyzer
+
+The browser timeframe analyzer discovers browser events that ocurred outside of the typical browsing window of this browser history.
+
+The analyzer determines the activity hours by finding the frequency of browsing events per hour, and then discovering the longest block of most active hours before proceeding with flagging all events outside of that time period. This information can be used by other analyzers or by manually looking for other activity within the inactive time period to find unusual actions.
+
+#### Chain alanyzer
+
+Sketch analyzer for chained events
+
+The purpose of the chain analyzer is to chain together events that can be described as linked, either by sharing some common entitites, or one event being a derivative of another event. An example of this would be that a browser downloads an executable, which then later gets executed. 
+
+The signs of execution could lie in multiple events, from different sources, but they are all linked or chained together. This could help an analyst see the connection between these separate but chained events. Another example could be a document written and then compressed into a ZIP file, which would then be exfilled through some means. If the document and the ZIP file are chained together it could be easier for the analyst to track the meaning of an exfil event involving the compressed file.
+
+#### Domain Analyzer
+
+The Domain Analyzer extracts domain and TLD info from events that have a field with either ```url``` or ```domain```.
+
+It will also add information about:
+
+- Known CDN providers (based on timeskcteh config)
+
+#### Windows EVTX Sessionizer
+
+This Analyzer will determine start end event for a user session based on EVTX events.
+
+#### Windows EVTX Gap analyzer
+
+It attempts to detect gaps in EVTX files found in an index  using two different methods.
+
+First of all it looks at missing entries in record numbers and secondly it attempts to look at gaps in days with no records.
+
+This may be an indication of someone clearing the logs, yet it may be an indication of something else. At least this should be interpreted as something that warrants a second look.
+
+This will obviously not catch every instance of someone clearing EVTX records, even if that's done in bulk. Therefore it should not be interpreted that if this analyzer does not discover something that the records have not been wiped. Please verify the results given by this analyzer.
```

Comment:
I would also add that the results of this analyzer is a story, that details the findings

Reply:
done